### PR TITLE
Feat/aut 3590/add metadata check on import

### PR DESCRIPTION
**Pull request by Karol-Stelmaczonek**
Ticket: https://oat-sa.atlassian.net/browse/AUT-3590

## What's Changed

- Add metametadata check & validation on import

## Dependencies PRs

- https://github.com/oat-sa/extension-tao-testqti/pull/2452
- https://github.com/oat-sa/extension-tao-itemqti/pull/2481

## How to test
- Create import with metadata
- Run import
- If import class don't have proper metadata the import will fail

### CODE REVIEW

**Review by bartlomiejmarszal — APPROVED** [review]
Remember to update composer.json before merge

File: models/classes/metadata/metaMetadata/PropertyMapper.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    {
+        return in_array($property->getUri(), $this->getIgnoredProperties());
+    }
+
+    private function getIgnoredProperties(): array
+    {
+        return [
+            OntologyRdf::RDF_TYPE,
+            taoTests_models_classes_TestsService::PROPERTY_TEST_TESTMODEL,
+            RDFS_LABEL
```
Comment (pnal) [inline models/classes/metadata/metaMetadata/PropertyMapper.php:75]: Can we avoid using global var?
```suggestion
            oat\generis\model\OntologyRdfs::RDFS_LABEL
```

File: models/classes/metadata/metaMetadata/PropertyMapper.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    private function isIgnoredForCollectionGathering(Property $property): bool
+    {
+        return in_array($property->getUri(), $this->getIgnoredProperties());
+    }
+
+    private function getIgnoredProperties(): array
+    {
+        return [
+            OntologyRdf::RDF_TYPE,
```
Comment (pnal) [inline models/classes/metadata/metaMetadata/PropertyMapper.php:73]: Does it make sense to add the list of ignored properties as a constructor param same as for `metaMetadataCollectionToExport`?

**Review by pnal — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by tikhanovichA — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**